libraries/default.rb

```diff
@@ -0,0 +1,15 @@
+def credential_secrets(databag = node['osl-git']['secrets_databag'], item = node['osl-git']['secrets_item'])
+  data_bag_item(
+    databag,
+    item
+  )
```

Comment:
The way this is laid out would require me to fragment my web1 databag into a separate databag for each git token, which I'm not sure is the way to go.

Reply:
I don't believe that's the case. Take a peek at the [example in the README](https://github.com/osuosl-cookbooks/osl-git/blob/b0ad1a5d11a341618a08bf394bfeebf6e271ecef/README.md#databag), it expects a list with key "credentials" containing the various credentials entries (each would be its own token).